src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react'
+import { Stack, Grid, Avatar, Typography, Button, Chip, Card, CardContent } from '@mui/material'
```

Comment:
Let me confirm that our vite tooling is set up for tree-shaking. The old advice was to import each component one by one to minimized the bundle size. Seems like the newer version allows us to do simpler imports like the one here.

https://mui.com/material-ui/guides/minimizing-bundle-size/#development-environment